test/suite/run.py

```diff
@@ -178,16 +178,38 @@ def show_env(verbose, envvar):
 # e.g. test_util03 -> util
 reCatname = re.compile(r"test_([^0-9]+)[0-9]*")
 
+# look for a list of the form 0-9,11,15-17
```

Comment:
Capitalize the start of comments.

Reply:
Fixed. Sorry, going to take a while to get used to that...